src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -59,7 +59,7 @@ internal static SafeSslHandle AllocateSslContext(SslProtocols protocols, SafeX50
                     throw CreateSslException(SR.net_allocate_ssl_context_failed);
                 }
 
-                Ssl.SetProtocolOptions(innerContext, protocols);
+                Ssl.SetProtocolOptions(innerContext.DangerousGetHandle(), protocols);
```

Comment:
> but it's not obvious why we aren't just passing the SafeHandle here.

Because the signature of SetProtcolOptions is being changed to take an IntPtr so that we don't need to artificially allocate a SafeHandle in HttpClient:
https://github.com/dotnet/corefx/pull/7485/files#diff-b18d4d400936e08d8a8fc68566a64f20R146

I could define two different P/Invoke signatures, one that takes a SafeHandle and one that takes an IntPtr, but given this usage is safe, didn't seem worth the extra baggage.